### PR TITLE
Update adding-forms.md

### PR DESCRIPTION
**Pull request by abhaypawar**

### CODE REVIEW

**Comment by abhaypawar** [comment]
Proofread and corrected major errors in translation thoroughly.
Please check @spielers @visshaljagtap.

Thanks again!

File: docs/docs/adding-forms.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default class IndexPage extends React.Component {
 }
 ```
 
-Now that you have created a class component, you can add `state` to the component.
+आता जेव्हा आपण क्लास कॉम्पोनन्ट तयार केला आहे तर आपण कॉम्पोनन्ट मध्ये `state` ऍड करू  शकतो.
```
Comment (spielers) [inline docs/docs/adding-forms.md]: remove the white spaces from karu shakta

File: docs/docs/adding-forms.md [hunk 2]
```diff
@@ -173,6 +173,7 @@ export default class IndexPage extends React.Component {
 }
 ```
 
-This form isn't doing anything besides showing the user information that they just entered. At this point, you may want to move this form to a component, send the form state to a backend server, or add robust validation. You can also use fantastic React form libraries like [Formik](https://github.com/jaredpalmer/formik) or [Final Form](https://github.com/final-form/react-final-form) to speed up your development process.
+हा फॉर्म वापरकर्त्याची माहिती दाखवण्याव्यतिरिक्त काही अतिरिक्त करत नाही. या वेळी, आपण हा फॉर्म एका कॉम्पोनन्ट कडे हलवू शकता, फॉर्म स्टेट बॅकएंड सर्वरवर पाठवा, किंवा रोबस्त मान्यता जोडा. तुम्ही अप्रतिम रिऍक्ट देखील वापरू शकता, जसे लायब्ररी [Formik](https://github.com/jaredpalmer/formik) किंवा [Final Form](https://github.com/final-form/react-final-form) आपल्या विकास प्रक्रियेला गती देण्यासाठी वापरू शकता.
```
Comment (spielers) [inline docs/docs/adding-forms.md]: मान्यता ==> validation it should be in English

**Review by spielers — CHANGES_REQUESTED** [review]
Hey, @abhaypawar Thanks for your contributions. have suggested few more changes please resolve that.
so I can merge this.

File: docs/docs/adding-forms.md (hunk 1 above)
Comment (abhaypawar) [reply docs/docs/adding-forms.md]: Done!

File: docs/docs/adding-forms.md (hunk 2 above)
Comment (abhaypawar) [reply docs/docs/adding-forms.md]: Corrected. Please check.
Thanks @spielers and please review - @visshaljagtap

**Review by spielers — APPROVED** [review]

**Pull request merged**